src/elements/container/container/container.scss

```diff
@@ -4,6 +4,8 @@
 @include sbb.box-sizing;
 
 :host {
+  --sbb-image-object-position: bottom;
```

Comment:
Is there an 'end' available or not?

Reply:
Nope, at least that's what I could gather from the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position)